datascience/README.md

```diff
@@ -52,3 +59,41 @@ Returns :
 3      DE      4229.0  biomass  2016-12-23T00:00:00+00:00
 4      DE     19919.0     coal  2016-12-23T00:00:00+00:00
 ```
+
+#### get_exchange(countries, start_date, end_date)
+
+This function returns for each day between `start_date` and `end_date`, the electricity production exchanges for each country defined in `countries`. Data are returned as a [pandas DataFrame](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html).
+
+Parameters :
+
+- countries (*list*) - a list of country codes to pull.
```

Comment:
@corradio - interesting! Do you mean a pair of country `country_from` and `country_to` ? 

I think the interesting point of that `get_exchange` is to return all `countries_to` of a `country_from`. You might not know beforehand what are the countries that exchange electricity with France for instance.